packages/components/src/components/Button/index.tsx

```diff
@@ -88,18 +101,17 @@ const commonStyles = {
   justifyContent: 'center',
   alignItems: 'center',
   flex: 'none', // as a flex child
-  cursor: 'pointer',
   fontFamily: 'Inter, sans-serif',
-  paddingY: 0,
+  paddingY: '4px',
+  paddingX: '8px',
   width: '100%',
-  paddingX: 2,
-  height: '26px', // match with inputs
-  fontSize: 2,
+  height: '28px', // match with inputs
+  fontSize: '13px',
   fontWeight: 'medium',
-  lineHeight: 1, // trust the height
+  lineHeight: '16px', // trust the height
   border: 'none',
-  borderRadius: 'small',
-  transition: 'all ease-in',
+  borderRadius: '4px',
+  transition: 'background .3s, color .3s',
   textDecoration: 'none',
   transitionDuration: theme => theme.speeds[2],
```

Comment:
Does the duration (`300ms`) defined in line 114 gets overridden by the transition duration (`theme.speeds[2] = 100ms`) set in line 116?